packages/frontend/src/pages/QuantifyPeriodReceiver/components/QuantifyMultipleButton.tsx

```diff
@@ -0,0 +1,29 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPrayingHands } from '@fortawesome/free-solid-svg-icons';
+
+interface Props {
+  disabled?: boolean;
+  onClick();
+}
+
+const QuantifyMultipleButton = ({
+  disabled = false,
+  onClick,
+}: Props): JSX.Element => {
+  return (
+    <button
+      disabled={disabled}
+      className={
+        disabled
+          ? 'praise-button-disabled space-x-2'
+          : 'praise-button space-x-2'
+      }
+      onClick={onClick}
+    >
+      <FontAwesomeIcon icon={faPrayingHands} size="1x" />
```

Comment:
What do you think about using the "unbalanced scale" icon instead of praying hands? https://fontawesome.com/search?q=scale%20unbalanced&s=solid%2Cbrands%2Cregular